Authentication/Login/View/LoginController.swift

```diff
@@ -251,26 +239,16 @@ extension LoginController {
         As both textfields fit in all devices, it will always show the email
         textfield at the top of the screen.
     */
-    func calculateTextFieldOffsetToMoveFrame(keyboardOffset: CGFloat) -> CGFloat {
+    private func calculateTextFieldOffsetToMoveFrame(keyboardOffset: CGFloat) -> CGFloat {
         return loginView.emailTextField.convertPoint(loginView.emailTextField.frame.origin, toView: self.view).y - 10
     }
     
-    func dismissKeyboard(sender: UITapGestureRecognizer) {
+    // Internal to avoid using @objc label for using it in selector.
+    internal func dismissKeyboard(sender: UITapGestureRecognizer) {
```

Comment:
this works? Having a protocol conformance in an extension? Anyway, you could move this to another extension and leave this extension as private. 
(Though I prefer to label this private and have the @objc)